From f427883320cbb65e254644717f5d7963e37bbd1d Mon Sep 17 00:00:00 2001
From: Steve Markgraf <steve@steve-m.de>
Date: Wed, 26 Feb 2020 23:52:02 +0100
Subject: [PATCH] rtl_tcp: Initialize listensocket

Older versions of GCC will complain that it can be used
uninitialized - which is not the case, but it breaks our Jenkins
build as we build with -Werror.
---
 src/rtl_tcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rtl_tcp.c b/src/rtl_tcp.c
index 42b90cb..562198f 100644
--- a/src/rtl_tcp.c
+++ b/src/rtl_tcp.c
@@ -394,7 +394,7 @@ int main(int argc, char **argv)
 	void *status;
 	struct timeval tv = {1,0};
 	struct linger ling = {1,0};
-	SOCKET listensocket;
+	SOCKET listensocket = 0;
 	socklen_t rlen;
 	fd_set readfds;
 	u_long blockmode = 1;
